jenkins/job-configs/kubernetes-jenkins-pull/fejta-pull.yaml

```diff
@@ -42,11 +42,10 @@
         - ansicolor:
             colormap: xterm
     builders:
-        - activate-gce-service-account
-        - shell: JENKINS_BUILD_STARTED=true "${{WORKSPACE}}/test-infra/jenkins/upload-to-gcs.sh"
         - shell: |
-            git clone https://github.com/kubernetes/test-infra
-            './test-infra/{cmd}'
+            # TODO(fejta): change this back to kubernetes/test-infra -b master
+            git clone https://github.com/fejta/test-infra -b fejta
+            './test-infra/jenkins/builder.sh' '{cmd}'
```

Comment:
Sounds like great feedback for when this system works and I am trying to switch something over to this system, but this bike-shedding seems like small-potatoes for this PR that unblocks me from doing further testing/development on a test job.